### PR TITLE
Image import

### PR DESCRIPTION
**Pull request by NaulaN**

### CODE REVIEW

File: src/ajax/send_image.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?php
+// require "../connexion.php";
+
+// session_start();
+// Connexion::set_up_connection("../../res/");
```
Comment (almerion) [inline src/ajax/send_image.php:5]:
```suggestion
require "../connexion.php";

session_start();
Connexion::set_up_connection("../../res/");
```

File: src/ajax/send_image.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    
+    // foreach ($data as $key) {
+    //     echo $key;
+    // }
+    // if (canBeSend($json)) {
+    //     $idCompte = $_SESSION['id'];
+    //     $idCategorie = $json->{"idCategorie"};
+    //     $title = $json->{"title"};
+    //     if (isset($idCompte, $idCategorie, $title)) {
+    //         $query = "INSERT INTO public.exercices VALUES (DEFAULT,:idCompte,:idCategorie,:title,:json)";
```
Comment (almerion) [inline src/ajax/send_image.php:26]: should be uncommented i think ?

File: src/css/home-page.css [hunk 3]
```diff
@@ -196,7 +197,7 @@ aside {
     margin-bottom: 5vh;
 }
 
-.titleAside > h1 {
+.titleAside>h1 {
```
Comment (almerion) [inline src/css/home-page.css:200]: css norm kebab-case

File: src/css/upload-Image.css [hunk 4]
```diff
@@ -124,7 +124,7 @@
     padding-top: 0;
 }
 
-#headerMenu > * {
+#headerMenu>* {
```
Comment (almerion) [inline src/css/upload-Image.css:127]: css norm kebab-case too (the whole file)

File: src/html/upload-Image.html [hunk 5]
```diff
@@ -37,12 +36,14 @@ <h1>Add Image Menu</h1>
                     <label for="image_uploads"><img src="https://cdn-icons-png.flaticon.com/512/32/32339.png"
                                                     width="20"></label>
                     <img id="remove" src="https://cdn-icons-png.flaticon.com/512/1828/1828901.png" width="20">
-                    <button>Add all picture</button>
+                    <button id="uploadImage">Add all picture</button>
```
Comment (almerion) [inline src/html/upload-Image.html:39]: css id kebab-case

File: src/index.php [hunk 6]
```diff
@@ -5,8 +5,8 @@
 require_once "./modules/mod_home/mod_home.php";
 require_once "./connexion.php";
 
-session_start();
-Connexion::set_up_connection();
+// session_start();
+// Connexion::set_up_connection();
```
Comment (almerion) [inline src/index.php:9]:
```suggestion
session_start();
Connexion::set_up_connection();
```

File: src/js/upload-Image.js [hunk 7]
```diff
@@ -10,143 +12,169 @@ const fileTypes = [
...
     "image/x-icon"
 ];
+const urlTypes = [
+    /.gif/,
+    /.jpeg/,
+    /.jpg/,
+    /.png/,
+];
 
+function getType() {
```
Comment (almerion) [inline src/js/upload-Image.js:22]: nothing in the function ?

File: src/js/show_hide_popIn_Image.js [hunk 8]
```diff
@@ -1,6 +1,21 @@
...
-function show() {
-    document.getElementById("pop-in_Image").style.display = 'block';
-};
\ No newline at end of file
+openMenuImage.onclick = function() {
+    if (iframeImgMenu.style.display == 'block')
+        iframeImgMenu.style.display = 'none';
+    else
+        iframeImgMenu.style.display = 'block';
+};
```
Comment (NaulaN) [inline src/js/show_hide_popIn_Image.js:10]:
```suggestion
openMenuImage.addEventListener("click", () => iframeImgMenu.style.display = (iframeImgMenu.style.display == 'block') ? "none" : "block");
```

File: src/js/show_hide_popIn_Image.js [hunk 9]
```diff
@@ -1,6 +1,21 @@
...
+
+iframeImgMenu.addEventListener('load', function() {
+    const exitMenu = iframeImgMenu.contentDocument.getElementById("exit");
+
+    exitMenu.onclick = function() {
+        if (iframeImgMenu.style.display == 'block')
+            iframeImgMenu.style.display = 'none';
+        else
+            iframeImgMenu.style.display = 'block';
+    }
```
Comment (NaulaN) [inline src/js/show_hide_popIn_Image.js:20]: Same code here, maybe is a good choice to create a function for this part
I seen at line 5 to line 10
And simplify it

**Review by NaulaN — COMMENTED** [review]
@MehediT look at my review and change it the duplicate of code

File: src/ajax/send_image.php (hunk 2 above)
Comment (MehediT) [reply src/ajax/send_image.php]: no, that is a example

**Pull request merged**